### PR TITLE
New package: TwitchNotifier-0.3.1

### PR DESCRIPTION
**Pull request by GiedriusS**

### CODE REVIEW

File: srcpkgs/TwitchNotifier/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'TwitchNotifier'
+pkgname="TwitchNotifier"
+version="0.3.1"
+revision=1
+build_style=python-module
+hostmakedepends="python3.4-devel"
+makedepends="python3.4-devel"
+depends="virtual?python3.4-gobject>=3.14.0_2 virtual?python3.4-requests>=2.7.0_1 virtual?libnotify>=0.7.6_4"
```
Comment (xtraeme) [inline srcpkgs/TwitchNotifier/template]: virtual is wrong here.

Comment (GiedriusS) [inline srcpkgs/TwitchNotifier/template]: Why? Those packages aren't needed for packaging up TwitchNotifier.

Comment (xtraeme) [inline srcpkgs/TwitchNotifier/template]: Also there's no reason to force a greater than or equal version.

**Comment by xtraeme** [comment]
`depends` only check if those pkgs are available as binary packages, so that they aren't installed to the masterdir, just verified and if are not available they are built.

`virtual?dep` is not meant to be used to skip this step.

**Comment by GiedriusS** [comment]
Should I remake the whole commit or could I just add a new commit on top of it with the changes that remove virtual and >=?

**Comment by xtraeme** [comment]
Up to you.

**Comment by xtraeme** [comment]
You are missing `noarch=yes`

**Comment by GiedriusS** [comment]
Sorry for causing so much issues.

**Pull request merged**